patches/common/chromium/.patches.yaml

```diff
@@ -459,3 +459,13 @@ patches:
   author: Jeremy Apthorp <jeremya@chromium.org>
   file: backport_cd7154e0bb5.patch
   description: Support macosx 10.14 SDK
+-
+  author: Shelley Vohr <shelley.vohr@gmail.com>
+  file: fix_xcode_ten.patch
+  description: |
+    Backports 4287a0d3, a commit which does the following -
```

Comment:
should note that this is the `build` commit id, the chromium commit hash is https://chromium.googlesource.com/chromium/src/+/27f9cbd88beea37bda9d6b46dabec9cf5eb386f3